script/RumpelConfig.sol

```diff
@@ -143,6 +143,16 @@ library RumpelConfig {
     address public constant MAINNET_YT_WSTUSR_26MAR2025 = 0xe0e034AfF49755e80b15594ce3A16d74d1a09b2F;
     address public constant MAINNET_YT_LVLUSD_28MAY2025 = 0x65901Ac9EFA7CdAf1Bdb4dbce4c53B151ae8d014;
     address public constant MAINNET_YT_EBTC_25JUNE2025 = 0x6a162ea0F31dC63Cd154f4fcCDD43B612Df731e9;
+    address public constant MAINNET_YT_SUSDE_30JUL2025 = 0xb7E51D15161C49C823f3951D579DEd61cD27272B;
+    address public constant MAINNET_YT_EUSDE_28MAY2025 = 0x708dD9B344dDc7842f44C7b90492CF0e1E3eb868;
+    address public constant MAINNET_YT_USDE_30JUL2025 = 0x733Ee9Ba88f16023146EbC965b7A1Da18a322464;
+    address public constant MAINNET_YT_LVLUSD_24SEP2025 = 0x946934554a2Bf59039661f971986F0223E906264;
+    address public constant MAINNET_YT_USR_28MAY2025 = 0x77DE4Be22Ecc633416D79371eF8e861Fb1d2cC39;
+    address public constant MAINNET_YT_WSTUSR_24SEP2025 = 0x1E24B022329f3CA0083b12FAF75d19639FAebF6f;
+    address public constant MAINNET_YT_WEETHS_25JUNE2025 = 0xaaC7DB6C2bC926aDE954D69A2d705f059043EA02;
```

Comment:
This one has a new SY token for us to add, I believe

Reply:
Added